packages/wrangler/src/dev.tsx

```diff
@@ -411,12 +414,15 @@ This is currently not supported 😭, but we think that we'll get it to work soo
 		} else {
 			assert(!(devInstance instanceof DevEnv));
 
-			watcher = devInstance.watcher;
+			configFileWatcher = devInstance.configFileWatcher;
+			assetsWatcher = devInstance.assetsWatcher;
+
 			const { waitUntilExit } = devInstance.devReactElement;
 			await waitUntilExit();
 		}
 	} finally {
-		await watcher?.close();
+		await configFileWatcher?.close();
+		await assetsWatcher?.close();
```

Comment:
If the first call rejects, the 2nd won't be called.

We should wrap these in Promise.allSettled

Reply:
✅